pkg/querier/querier.go

```diff
@@ -39,6 +39,7 @@ type Config struct {
 	IngesterStreaming    bool          `yaml:"ingester_streaming"`
 	MaxSamples           int           `yaml:"max_samples"`
 	QueryIngestersWithin time.Duration `yaml:"query_ingesters_within"`
+	QueryStoreForLabels  bool          `yaml:"query_store_for_labels"`
```

Comment:
YAML config options must match CLI flags.

```suggestion
	QueryStoreForLabels  bool          `yaml:"query_store_for_labels_enabled"`
```